### ISSUE TITLE
Road Bases for A-10's and RW

### ISSUE THREAD

**Issue by 000Ready000** [132nd-vWing/TRMA#21]
Propose to add a couple of pop-up roadbases for Hogs (and RW?) to rearm/refuel near the AG ranges further away from Bodo. It is a dissapointment when we have to RTB and take a long hike because we spent all ordnance during missions while we still have plenty of fuel left. Willing to help find spots and setting up slides/procedures.

**Comment by victrox-dev** [comment]
Agreed. Happy to help find locations on the RW FARPs. Not saying aircraft spawns, but definitely some basic FARP locations. For us on the RW side of things, it would be nice to have a FARP on the border of RW Range 1 and 2, then another in RW Range 3. It's not reasonably feasible for us to return to ENBO during gunnery operations. Likewise, I'm happy to script the entire FARP for late activation, this way it's not "in the picture" when it's not necessarily being used.

-Sly

**Comment by 132ndNeck** [comment]
Sounds good, and feel free to post the suggestion/inputs here for implementation.
